src/app/sidebar/sidebar.component.html

```diff
@@ -10,19 +10,19 @@
             <i class="menu-icon fas fa-align-left"></i>
             <span>Offers</span>
         </li>
-        <li routerLink="my_portfolio" routerLinkActive="side-active">
+        <li routerLink="my_portfolio" routerLinkActive="side-active" [ngClass]="{'unverified-user-disabled': !hasVerifiedProfile}">
```

Comment:
You can use the syntax `[class.unverified-user-disabled]="!hasVerifiedProfile"` which is shorter and looks much cleaner.